ui/src/Pages/Preferences/Account/DelAccountBtn.jsx

```diff
@@ -25,7 +25,7 @@ function DelAccountBtn() {
     (async () => {
       if (deleteAccount.deleted && !deleteAccount.error) {
         await dispatch(logout());
-        history.push("/login");
+        window.location.href = "/";
```

Comment:
Is this a good idea? By using window.location.href we might break the react router

Reply:
What I was trying to avoid was pushing a user back to the login page in the event that they may have just deleted the only user account. So they should be sent back to the registration page in that case. And since that logic was already added to `PrivateRoute` I figured I would just trigger a page load to the root to hit that logic rather than replicating it.

Pushing to the history won't work in this case because while logged in they are already at the `/` route.

I admit this is a bit of a hack, but it worked well for me.